.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: go
+
+go:
+  - 1.4.1
```

Comment:
@franklinwise First of all, thanks for doing this! Yeah, just do `make test`, I'd say. It has the advantage that if we update the Go version in the `Makefile.COMMON` in the future, it'll automatically propagate to all the Go repos when they get the new `Makefile.COMMON`. Also, when we decide to vendor dependencies (which we probably will at some point), it will automatically do the right thing.